### PR TITLE
Fixed white space below Navbar when viewing below 480px

### PR DESCRIPTION
**Pull request by JessieVela**
Body had a padding of 50px which is more big than the nav bar which has a height of 49.188px. Made the padding of the body match the nav height.
issue #16

### CODE REVIEW

File: src/stylesheets/global.scss [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 body {
     margin: 0;
-    padding:  55px 0 0 0 ;
+    padding:  49.188px 0 0 0 ;
```
Comment (Polarts) [inline src/stylesheets/global.scss]: 0.188px is a little too specific...
It does work with 50px tho, so that would be more readable

Comment (Polarts) [reply src/stylesheets/global.scss]: Also, I'd love to know how you came up with this number. Did you use an editor?

Comment (JessieVela) [reply src/stylesheets/global.scss]: 50px will still have a white line if you go smaller like 320px.
Screen Shot 2020-03-05 at 12 02 16 PM
If you want readable then change the navbar height to be 50px and body 50px will resolve the line and keep the values readable.

Comment (Polarts) [reply src/stylesheets/global.scss]: Let's go with your suggestion then, it sounds much better than leaving it at 49.188px. Make the change and I'll merge your PR

Comment (JessieVela) [reply src/stylesheets/global.scss]: Google chrome inspector. It shows you the dimensions of elements. That's how I saw the navbar height is exactly 49.188px at 320px. Only problem is I have to add .812 of padding to the navbar.

Okay, I think I found a better solution. Pushing changes to review.

**Comment by JessieVela** [comment]
New changes pushed. I removed the transparent border as it served no purpose since the navbar should be flush with the body.

**Pull request merged**